src/test/aws/common/cdkConfig/pipes.json

```diff
@@ -2,6 +2,7 @@
   "testSqsToLambdaPipe": {
     "name": "test-sqs-to-lambda-pipe",
     "sqsBatchSize": 10,
+    "lambdaInputTemplate": "<$.body>",
```

Comment:
need to validate this in the test for pipe

Reply:
done